src/test/java/org/fcrepo/importexport/exporter/ExporterTest.java

```diff
@@ -370,53 +373,6 @@ public void testExportBagCustomTags() throws Exception, FcrepoOperationFailedExc
         Assert.assertTrue(customLines.contains("Foo : Bar"));
         Assert.assertTrue(customLines.contains("Baz : Quux"));
     }
-
-    @Test
-    public void testExportVersionsContainers() throws Exception {
```

Comment:
Although it looks like the removal of this test method is associated with the addition of:
https://github.com/fcrepo4-labs/fcrepo-import-export/pull/101/files#diff-3dc91ed69dd47e72f6396a8309355095R296

However, they are different. Can you speak to why this one needs to go away?